Day-I-Social-Card/social-card/src/components/CardComponents/CardContent.js

```diff
@@ -1,2 +1,19 @@
-import React from 'react';
-import './Card.css';
+import React from "react";
+import "./Card.css";
+
+const CardContent = () => {
+  return (
+    <div class="card-content">
+      <h3>Get Started with React</h3>
+      <p>
+        React makes it painless to create interactive UIs. Design simple views
+        for each state of your application.
+      </p>
+      <a className="card-link" href="https://react.js.org/">
```

Comment:
You're nesting a tags here (because the entire cardcontent is also wrapped in an a tag). You're probably getting an error message in your console log from React saying something about it.